docs/account/slack/slack-notifications.md

```diff
@@ -54,6 +52,24 @@ Keep scrolling down on Appcircle to see the full list of events.
 
 :::
 
+<Screenshot url='https://cdn.appcircle.io/docs/assets/image (136).png' />
+
+:::info
+After completing the specified action in Appcircle, you have the option to share release notes via Slack. 
+To enable this feature, ensure you include the [**Publish Release Notes**](https://docs.appcircle.io/integrations/managing-release-notes/) step in your workflow.
+
+Additionally, note that you can access download links for the release notes for a duration of 90 days.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-ReleaseNotesViaEmail.png' />
+
+:::info
+After completing the specified action in Appcircle, you have the option to share the test results via Slack.
+To enable this feature, ensure you include the [**Test Reports**](https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests#generating-test-report) step in your workflow.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-TestReportsViaEmail.png' />
```

Comment:
@csonuryilmaz @atkncamurcu The warning message "Cannot Connect to server" appears in the screen shot background. The screenshot may need to be changed. If it is not necessary, I fixed some typos and added related documentation links to mentioned workflow components.

@csonuryilmaz apart of these, ss links are good, template is good. fyi.